GSDemo/app/sampledata/cars.xml

```diff
@@ -0,0 +1,3654 @@
+<?xml version="1.0"?>
```

Comment:
Ae these Haar-Casacades trained by you or is there someone who claims ownership of these files?